tests/ui/requirements.txt

```diff
@@ -1,9 +1,51 @@
-fxapom==1.9.1
-PyJWT==1.4.2
-PyPOM==1.1.1
-pytest==3.0.6
-pytest-instafail==0.3.0
-pytest-selenium==1.9.1
-pytest-variables==1.5.1
-pytest-xdist==1.15.0
-selenium==3.0.2
+django-cache-machine==0.9.1 \
```

Comment:
They still need to get removed, don't they? Generally, this file should contain only the uttermost needed dependencies required for running the ui-tests **on top of installing ``dev.txt``** before